src/nordjylland_news/utils.py

```diff
@@ -0,0 +1,62 @@
+import jsonlines
+from bs4 import BeautifulSoup
+
+
+def init_jsonl(file_name: str):
+    """Initializes jsonl file
+
+    Args:
+        file_name (str):
+            File name to initialize
+    """
+    with open(file_name, "w") as _:
+        pass
+
+
+def append_jsonl(data: list, file_name: str):
+    """Appends data to jsonl file
+
+    Args:
+        data (list):
+            Data to append
```

Comment:
Mangler `file_name`.
```suggestion
    Args:
        data (list):
            Data to append
        file_name (str):
            The name of the JSONL file where the data should be appended.
```